mu-plugins/blocks/global-header-footer/postcss/header/search.pcss

```diff
@@ -91,27 +96,49 @@
 
 		& .wp-block-search__input {
 			width: 100%;
-			padding-right: 35px; /* Prevent user input from running into icon. */
+			padding-right: 50px; /* Prevent user input from running into icon. */
 			border: none;
+			border-radius: 2px;
 			background-color: var(--wp--preset--color--white);
-			color: var(--wp--preset--color--dark-grey);
+			color: var(--wp--preset--color--black);
 			font-size: inherit;
+			-webkit-appearance: none;
```

Comment:
Is this what gets rid of Safari-mobile's extra glass icon? If so, a comment seems useful to make that obvious.